rbac/internal/views.py

```diff
@@ -1447,6 +1471,48 @@ def retrieve_ungrouped_workspace(request):
         return HttpResponse(str(e), status=500)
 
 
+def lookup_resource(request):
+    """POST to retrieve resource details from relations api."""
+    # Parse JSON data from the POST request body
+    req_data = json.loads(request.body)
+
+    # Request parameters for resource lookup on relations api from post request
+    resource_type_name = req_data["resource_type"]["name"]
+    resource_type_namespace = req_data["resource_type"]["namespace"]
+    resource_subject_name = req_data["subject"]["subject"]["type"]["name"]
+    resource_subject_id = req_data["subject"]["subject"]["id"]
+
+    try:
+        with create_client_channel(settings.RELATION_API_SERVER) as channel:
+            stub = lookup_pb2_grpc.KesselLookupServiceStub(channel)
+
+            request = lookup_pb2.LookupResourcesRequest(
+                resource_type=common_pb2.ObjectType(
+                    name=resource_type_name,
+                    namespace=resource_type_namespace,
+                ),
+                relation="member",
+                subject=common_pb2.SubjectReference(
+                    subject=common_pb2.ObjectReference(
+                        type=common_pb2.ObjectType(namespace=resource_type_namespace, name=resource_subject_name),
+                        id=resource_subject_id,
+                    ),
+                ),
+            )
+        responses = stub.LookupResources(request)
+
+        if responses:
+            for r in responses:
+                return HttpResponse(r.resource)
```

Comment:
it would be great if we would have response in json format(you can useJsonResponse)